SeleniumPracticalExercise/PageObjects/EmployeeInformationPage.cs

```diff
@@ -6,19 +6,20 @@ namespace SeleniumPracticalExercise.PageObjects
 {
     public class EmployeeInformationPage : BasePageLocal
     {
-        
+        private readonly By _actualFirstNameLocator = By.XPath("//div[contains(@class,'oxd-table-row--clickable')]//div[text()=''][3]/div");
+        private readonly By _actualIDLocator = By.XPath("//div[contains(@class,'oxd-table-row--clickable')]//div[text()=''][2]/div");
+        private readonly By _actualLastNameLocator = By.XPath("//div[contains(@class,'oxd-table-row--clickable')]//div[text()=''][4]/div");
         private readonly By _addButtonLocator = By.CssSelector("div.orangehrm-header-container > button");
-        private readonly By _employeeIDTxtBoxLocator = By.CssSelector("div.oxd-input-group input.oxd-input");  //EmployeeInfoPage
-        private readonly By _expectedIDLocator = By.XPath("//div[contains(@class,'oxd-table-row--clickable')]//div[text()=''][2]/div");
-        private readonly By _expectedFirstNameLocator = By.XPath("//div[contains(@class,'oxd-table-row--clickable')]//div[text()=''][3]/div");
-        private readonly By _expectedLastNameLocator = By.XPath("//div[contains(@class,'oxd-table-row--clickable')]//div[text()=''][4]/div");
+        private readonly By _employeeIDTxtBoxLocator = By.CssSelector("div.oxd-input-group input.oxd-input"); 
```

Comment:
We talked in general about not using abbreviations, e.g. "Txt".

In this case, use "Field" instead of "TxtBox"... so "_employeeIdFieldLocator". We haven't talked about that I don't think? It's a automation team preference just to be consistent.

Reply:
rename to have Field instead of textbox